sdk/src/metrics/aggregation/histogram_aggregation.cc

```diff
@@ -96,20 +104,27 @@ void DoubleHistogramAggregation::Aggregate(double value, const PointAttributes &
 std::unique_ptr<Aggregation> DoubleHistogramAggregation::Merge(
     const Aggregation &delta) const noexcept
 {
-  // TODO - Implement me
-  return nullptr;
+  auto curr_value  = nostd::get<HistogramPointData>(ToPoint());
+  auto delta_value = nostd::get<HistogramPointData>(
+      (static_cast<const DoubleHistogramAggregation &>(delta).ToPoint()));
+  DoubleHistogramAggregation *aggr = new DoubleHistogramAggregation();
+  HistogramMerge<double>(curr_value, delta_value, aggr->point_data_);
+  return std::unique_ptr<Aggregation>(aggr);
 }
 
 std::unique_ptr<Aggregation> DoubleHistogramAggregation::Diff(
     const Aggregation &next) const noexcept
 {
-  // TODO - Implement me
-  return nullptr;
+  auto curr_value = nostd::get<HistogramPointData>(ToPoint());
+  auto next_value = nostd::get<HistogramPointData>(
+      (static_cast<const DoubleHistogramAggregation &>(next).ToPoint()));
+  DoubleHistogramAggregation *aggr = new DoubleHistogramAggregation();
+  HistogramDiff<double>(curr_value, next_value, aggr->point_data_);
+  return std::unique_ptr<Aggregation>(aggr);
 }
 
 PointType DoubleHistogramAggregation::ToPoint() const noexcept
```

Comment:
Do you think we need to change the name of this?

Reply:
Changing method name from `ToPoint()`?  The method is supposed to give PointData, which is then converted into MetricData during aggregation. Do you have suggestion for better naming?

Reply:
`ToPoint` sounds like we are converting to point data. How about `GetPoint`?

Reply:
Yes, good point. I was doing conversion in the initial implementation, was changed later. Should we do it as separate issue, as this will have changes in quite a few other files not related to this PR?

Reply:
yeah let's do it in a separate PR.